### PR TITLE
Windows timestamp

### PR DESCRIPTION
**Pull request by pawelmhm**
This is just same as #43 expects it:

* adds unit tests 
* adds way to customize log file timeformat
* goes with format  that contains dots so for example `2016-12-22T145616.839288.log`

cc @zolrath

### CODE REVIEW

**Comment by redapple** [comment]
Is the _dot_ between minutes and seconds necessary?
Without it, the string becomes ISO 8601 compatible

```
>>> import iso8601
>>> iso8601.parse_date("2016-12-22T1456.16.839288")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/paul/.virtualenvs/isodate/local/lib/python2.7/site-packages/iso8601/iso8601.py", line 190, in parse_date
    raise ParseError("Unable to parse date string %r" % datestring)
iso8601.iso8601.ParseError: Unable to parse date string '2016-12-22T1456.16.839288'
>>> iso8601.parse_date("2016-12-22T145616.839288")
datetime.datetime(2016, 12, 22, 14, 56, 16, 839288, tzinfo=<iso8601.Utc>)
```

**Comment by pawelmhm** [comment]
ah good point @redapple I'll fix that

File: scrapyrt/conf/default_settings.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 # Path to server log file
 LOG_FILE = None
+SPIDER_LOGFILE_TIMEFORMAT = '%Y-%m-%dT%H%M%S.%f'
```
Comment (chekunkov) [inline scrapyrt/conf/default_settings.py]: `LOG_FILE_TIME_FORMAT` to be more consistent with existing variable names?

Comment (pawelmhm) [inline scrapyrt/conf/default_settings.py]: LOG_FILE is server file. SPIDER_LOG_FILE is file where spider logs will be placed. So I think keeping SPIDER prefix is ok, but yeah I added comment and changed LOGFILE to LOG_FILE.

**Comment by chekunkov** [comment]
@pawelmhm aside from https://github.com/scrapinghub/scrapyrt/pull/47#discussion_r94612409 looks good, feel free to merge after you fix it.

**Pull request merged**